.gitignore

```diff
@@ -1,3 +1,3 @@
 /twittermost
-/twittermost.json
-/config.json
+*/mattermost.json
```

Comment:
Ah, yeah, so I wrote `mattermost.json` instead of `twittermost.json`.  ^^'

Reply:
yep